packages/core/plugins/core-plugins-server-internal/src/plugin.test.ts

```diff
@@ -17,15 +17,16 @@ import type { CoreContext } from '@kbn/core-base-server-internal';
 import { loggingSystemMock } from '@kbn/core-logging-server-mocks';
 import type { NodeInfo } from '@kbn/core-node-server';
 import { nodeServiceMock } from '@kbn/core-node-server-mocks';
-import { coreMock } from '../mocks';
-
 import { PluginWrapper } from './plugin';
-import { PluginManifest, PluginType } from './types';
+
+import type { PluginManifest } from '@kbn/core-plugins-server';
```

Comment:
nit: absolute imports should be placed before relative imports (for some reason the ES-lint rule don't apply to the `.test.ts` files 🤷

Reply:
@elastic/kibana-operations is there a reason we're excluding `*.test.ts` files from the import-order ES-lint rule?